1st-gen/tools/truncated/README.md

```diff
@@ -1,49 +1,241 @@
 
-It is used like a `<span>` to contain potentially-long strings that are important for users to see, even when in small containers, like full
-names and email addresses.
+This component is designed like a `<span>` to contain potentially-long strings that are important for users to see, even when in small containers, such as full names, email addresses, file paths, and other text data that may exceed the available space.
 
 ### Usage
 
 [![See it on NPM!](https://img.shields.io/npm/v/@spectrum-web-components/truncated?style=for-the-badge)](https://www.npmjs.com/package/@spectrum-web-components/truncated)
 [![How big is this package in your project?](https://img.shields.io/bundlephobia/minzip/@spectrum-web-components/truncated?style=for-the-badge)](https://bundlephobia.com/result?p=@spectrum-web-components/truncated)
 
-```
+```bash
 yarn add @spectrum-web-components/truncated
 ```
 
-Import the side effectful registration of `<sp-truncated>` via:
+Import the side effectful registration of `<sp-truncated>` as follows:
 
-```
+```typescript
 import '@spectrum-web-components/truncated/sp-truncated.js';
 ```
 
 When looking to leverage the `Truncated` base class as a type and/or for extension purposes, do so via:
 
-```
+```typescript
 import { Truncated } from '@spectrum-web-components/truncated';
 ```
 
-## Example
+### Anatomy
+
+The basic structure of the truncated component:
 
 ```html
 <sp-truncated>
     This will truncate into a tooltip if there isn't enough space for it.
 </sp-truncated>
 ```
 
-### With specific overflow content
+### Examples
+
+#### Basic usage
+
+By default, `<sp-truncated>` will automatically detect when text overflows its container and display a tooltip with the full content on hover. When the truncated text is clicked, it copies the full text to the clipboard.
+
+```html demo
+<div
+    style="width: 200px; border: 1px solid var(--spectrum-gray-300); padding: 8px; overflow: hidden; resize: both;"
+>
+    <sp-truncated>
+        This is a very long sentence that should be truncated when there isn't
+        enough space to display it fully.
+    </sp-truncated>
+</div>
+```
+
+#### With mixed content
+
+`<sp-truncated>` supports mixed content including text formatting and inline elements:
+
+```html demo
+<div
+    style="width: 250px; border: 1px solid var(--spectrum-gray-300); padding: 8px;"
+>
+    <sp-truncated>
+        This is a
+        <strong>very long</strong>
+        sentence with
+        <em>formatted text</em>
+        that should be truncated.
+    </sp-truncated>
+</div>
+```
+
+#### Long words without spaces
+
+The component handles long words or strings without spaces appropriately:
+
+```html demo
+<div
+    style="width: 150px; border: 1px solid var(--spectrum-gray-300); padding: 8px;"
+>
+    <sp-truncated>
+        ThisIsAVeryLongWordWithoutAnySpacesThatShouldBeTruncated
+    </sp-truncated>
```

Comment:
This example isn't working on the docs page as the tooltip does not show the entire content.